easybuild/easyconfigs/c/Crass/Crass-0.3.12-goolf-1.7.20.eb

```diff
@@ -0,0 +1,34 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock = 'ConfigureMake'
+
+name = 'Crass'
+version = '0.3.12'
+
+homepage = 'http://bioinformatics.ninja/crass/'
+description = """ Crass is a program that searches through raw metagenomic reads 
+ for Clustered Regularly Interspersed Short Palindromic Repeats """
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+
+source_urls = ['https://github.com/ctSkennerton/crass/archive/']
+sources = ['v%(version)s.tar.gz']
+
+dependencies = [
+    ('Xerces-C++', '3.1.1'),
+    ('zlib', '1.2.8'),
+]
+
+preconfigopts = './autogen.sh && '
```

Comment:
this implies having `Autoconf`, `Automake`, `libtool` available as build dependencies

consider using the `Autotools` bundle defined by @wpoely86, see https://github.com/hpcugent/easybuild-easyconfigs/tree/develop/easybuild/easyconfigs/a/Autotools